JavaScript/Markdown_to_Html/script.js

```diff
@@ -0,0 +1,38 @@
+var output = '';
+inputTextarea = document.querySelector('#inputTextarea');
+outputTextarea = document.querySelector('#outputTextarea');
+buttonClear = document.querySelector('#buttonClear');
+buttonCopyHTML = document.querySelector('#buttonCopyHTML');
+buttonCopyMarkdown = document.querySelector('#buttonCopyMarkdown');
+buttonPreview = document.querySelector('#buttonPreview');
+
```

Comment:
As an enhancement, can you make UDFs to enable more structured code- based on the operations? In a nutshell, the idea is to remove all global vars.

Reply:
@sohamsshah  i have done all additions mentioned above.